src/main/groovy/com/ullink/testtools/elastic/TestExportTask.groovy

```diff
@@ -171,15 +202,18 @@ class TestExportTask extends Exec {
                 result.with {
                     failureMessage = node.@message
                     failureType = node.@type
-                    failureText = node.text()
                     resultType = TestResult.ResultType.FAILURE
                 }
             }
             if (it.name() == 'skipped') {
                 result.resultType = TestResult.ResultType.SKIPPED
             }
         }
+        result.feature = getEnrichment().resolveFeature(p, solution)
+        result.steps =  getEnrichment().resolveSteps(p,solution)
         result.properties = resolveProperties(p)
+        result.projectName= project.getName()
+
```

Comment:
same here, it should be done in enrichment
I can work with you to see how it can be done